### PR TITLE
Update and rename builders-program.md to alpha-program.md

### PR DESCRIPTION
**Pull request by joshuacheong**
Changed the Substrate Builders Program and Square One into the updated Polkadot Alpha Program (Please see the blog post for more context: https://polkadot.network/blog/the-polkadot-alpha-program-a-new-era-for-decentralized-building-collaboration/)

### CODE REVIEW

File: docs/general/alpha-program.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+id: alpha-program
+title: Polkadot Alpha Program
+sidebar_label: Alpha Program
+description: The Polkadot Alpha Program and How it can Support your Development.
+slug: ../alpha-program
+---
+
+Over 150 projects are using [Polkadot SDK known as Substrate](https://substrate.io) to create the next generation of
```
Comment (CrackTheCode016) [inline docs/general/alpha-program.md]:
```suggestion
Over 150 projects are using the [Polkadot SDK (also known as Substrate)](https://substrate.io) to create the next generation of
```

**Review by CrackTheCode016 — CHANGES_REQUESTED** [review]
looks mostly good to me, except we need to make sure we replace the sidebar item as well (in both polkadot-wiki and kusama).  this can be found in `polkadot-wiki/sidebars.js` and `kusama-guide/sidebars.js`

`"general/builders-program",` -> `"general/alpha-program"`,

**Comment by NM2146** [comment]
Hey @filippoweb3 Would it be possible to merge and make these changes as SBP & Square One no longer exists. Thanks :)

**Comment by CrackTheCode016** [comment]
@NM2146 Hey, sorry for the delay!  I was waiting on @joshuacheong to add the page to the sidebars file to ensure the wiki doesn't crash, as the page for SBP is deleted in this PR. Otherwise, it's good to merge.

 If this is too much trouble, I can insert it myself, let me know!

**Comment by filippoweb3** [comment]
@NM2146 I will need to add redirects as well. This can be done on a second PR. Let me merge this and do the changes in another PR. Need also to check for broken links.

**Review by filippoweb3 — APPROVED** [review]

**Comment by NM2146** [comment]
Thanks @filippoweb3 ! Will check for other links and create a new PR for re-directs :)

**Comment by filippoweb3** [comment]
@NM2146 no need we already did it yesterday. All good ;)

**Pull request merged**